### DISCUSSION TITLE
Changing /nsm from LVM to RAID1

### DISCUSSION THREAD

**Discussion by pimpernoni** [Security-Onion-Solutions/securityonion#6661] [Unsupported Versions]
I have loaded a fresh version of 2.3.90 onto a Standalone box with a 2-disk RAID1 already created. During the install, the RAID1 did not show up, just the component physical drives comprising the array. Prior users discovered this issue as well and were instructed to load the /nsm onto same drive where the /root is, and subsequently change the mount to the RAID. 

Followed these instructions: https://docs.securityonion.net/en/2.3/new-disk.html?highlight=adding%20a%20disk, Method 2.

However, when I try to #:mv /nsm /nsm.old, I get "mv: cannot move ‘/nsm’ to ‘/nsm.old’: Device or resource busy".
I suspect this has to do with the LVM formatting of the /dev/mapper/system-nsm seen when I check #: mount.

I would rather use RAID than LVM at this time, and I successfully did this redirect to the RAID on previous SO installs (2.3.30 ish). How can I effect the change from the current /nsm folder to the new RAID1? 

This is a Standalone install downloaded from SO-iso CentOS onto PC box with Intel i5 3200 Hz, 32 G RAM, 256 G nvme; 1 TB RAID1

Thanks

P

**Comment by dougburks** [comment]
`Device or resource busy` sounds like some service is using `/nsm`. Did you follow these steps to disable all services and reboot?
```
sudo systemctl disable salt-minion
sudo reboot
```

**Comment by pimpernoni** [comment]
Yes, stopped salt-minion, reboot and shutdown Docker after restart.
Thanks for any tips

P

**Comment by dougburks** [comment]
Sounds like something is still using `/nsm`. Have you installed any other software or made any other modifications?

After stopping salt-minion, rebooting, and shutting down Docker, what is the output of the following?
```
sudo lsof /nsm
```

**Comment by Masaya-A** [comment]
Unmount `/nsm`, make `/nsm.old` directory and then mount `/dev/mapper/system-nsm` as `/nsm.old`.
Or following commands may work.
```
sudo mkdir /nsm.old
sudo sed -i /etc/fstab -e "s/ \/nsm / \/nsm.old /g"
sudo systemctl disable salt-minion
sudo reboot
```
